x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/chart_section.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  Axis,
+  Settings,
+  Position,
+  Chart,
+  PointerUpdateListener,
+  TickFormatter,
+  TooltipValue,
+  ChartSizeArray,
+} from '@elastic/charts';
+import React from 'react';
+import moment from 'moment';
+import { MetricsExplorerSeries } from '../../../../../../../../common/http_api';
+import { MetricExplorerSeriesChart } from '../../../../../metrics_explorer/components/series_chart';
+import {
+  MetricsExplorerChartType,
+  MetricsExplorerOptionsMetric,
+} from '../../../../../metrics_explorer/hooks/use_metrics_explorer_options';
+import { ChartHeader } from './chart_header';
+import { getTimelineChartTheme } from '../../../../../metrics_explorer/components/helpers/get_chart_theme';
+
+const CHART_SIZE: ChartSizeArray = ['100%', 160];
+
+interface Props {
+  title: string;
+  style: MetricsExplorerChartType;
+  chartRef: React.Ref<Chart>;
+  series: ChartSectionSeries[];
+  tickFormatterForTime: TickFormatter<any>;
+  tickFormatter: TickFormatter<any>;
+  onPointerUpdate: PointerUpdateListener;
+  domain: { max: number; min: number };
+  isDarkMode: boolean;
+  stack?: boolean;
+}
+
+export interface ChartSectionSeries {
+  metric: MetricsExplorerOptionsMetric;
+  series: MetricsExplorerSeries;
+}
+
+export const ChartSection = ({
+  title,
+  style,
+  chartRef,
+  series,
+  tickFormatterForTime,
+  tickFormatter,
+  onPointerUpdate,
+  domain,
+  isDarkMode,
```

Comment:
I think we could avoid passing this down as a prop, since it doesn't seem to be used in the parent. What do you think?

Reply:
Nice catch. Makes sense.